src/layout/header/ChangeLanguageButton.tsx

```diff
@@ -34,14 +34,15 @@ const ChangeLanguageButton: FC = () => {
         startIcon={<LanguageIcon />}
         endIcon={open ? <ExpandLessIcon /> : <ExpandMoreIcon />}
         onClick={handleClick}>
-        {i18n.language}
+        {t('localization.language')}
```

Comment:
Isteden for å skrive "norsk" hvis man allerede er på norsk side eller "english" om man er på engelsk, synes jeg det var like greit å skrive funksjonen isteden.

Reply:
Så lenge brukere skjønner hvor de skal trykke for å endre språk er det jo greit.